### DISCUSSION TITLE
AEWeb - Add publication status (AEIP-13 )

### DISCUSSION THREAD

**Discussion by samuelmanzanera** [archethic-foundation/aeip#42] [AEIP]
This is a discussion for the AEIP13 located at: https://github.com/archethic-foundation/aeip/blob/main/AEIP-13.md

**Comment by samuelmanzanera** [comment]
Thanks @bchamagne  for the AEIP.

Could you include also a proper example of the new format of AEWeb tx reference with publication's status ?

**Comment by redDwarf03** [comment]
In the new schema, shouldn't the "PUBLISHED" value be added to the ``publicationStatus`` enum?

**Comment by Neylix** [comment]
Since AEWeb reference transaction uses JSON format, it should also require [AEIP-8](https://github.com/archethic-foundation/aeip/blob/main/AEIP-08.md) as it is mentionned to use it if possible

> This standard is not applicable only for token specification like AEIP-2 but can be used for all kind of standard that uses a JSON format.
It is recommended that future AEIP require this AEIP-8 if possible.

Applications should then implement the AEIP-8 in their transaction (AEWeb CLI and Web for instance)

**Comment by bchamagne** [comment]
Should we allow clients applications to send `publicationStatus = UNPUBLISHED`  but with lots of files still in the reference transaction? The transaction will be more expensive but when the website is published again, it doesn't have to upload everything  again, but only the diff.

**Reply by samuelmanzanera** [reply]
The reference transaction hold the addresses of other data chains, so the cost of the transaction will remain the same.
The thing is, in this scenario, the user will pay for files which would not be accessible.
But you are right, later the cost will be low to activate the website.

**Comment by bchamagne** [comment]
You'll find the updated proposal that takes your feedback into account: https://github.com/archethic-foundation/aeip/pull/51
